lib.Makefile

```diff
@@ -199,7 +199,7 @@ ifeq ($(BUILDARCH),amd64)
 	# *-amd64 tagged images for etcd are not available until v3.5.0
 	ETCD_IMAGE = quay.io/coreos/etcd:$(ETCD_VERSION)
 endif
-UBI_IMAGE ?= registry.access.redhat.com/ubi8/ubi-minimal:$(UBI_VERSION)
+UBI_IMAGE ?= registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
We do `microdnf upgrade` when building the container image. It essentially moves the tag to `latest`. We are not really pinning to the `UBI_VERSION` defined in `metadata.mk`.